client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/site-preview/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { useTranslate } from 'i18n-calypso';
+import React, { FC } from 'react';
+import { useQuery } from 'calypso/landing/stepper/hooks/use-query';
+import './style.scss';
+
+export const SitePreview: FC = () => {
+	const translate = useTranslate();
+	const fromUrl = useQuery().get( 'from' ) || '';
+
+	return (
+		<div className="migration-instructions-from-preview">
+			<iframe
+				className="migration-instructions-from-preview__iframe"
+				title={ translate( 'Preview of the site being imported' ) }
+				src={ fromUrl }
+			/>
```

Comment:
Should we have a fallback in case the site can't be loaded? Aren't there a few ways to prevent sites from being embedded?